### DISCUSSION TITLE
[React] Figma menu style with useClientPoint

### DISCUSSION THREAD

**Discussion by joaom00** [floating-ui/floating-ui#2290] [Q&A] [answered]
Hey guys!

Can I use `useClientPoint` to build something similar to this?

https://user-images.githubusercontent.com/48808846/232378869-bb0d7f70-9403-4d7e-b371-28704e6edea3.mp4

**Comment by atomiks** [comment, marked as answer]
You would need to make a custom drag hook, but the client point positioning should work by manually specifying the `x` and `y` client coordinates
